__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require("../app");
+const db = require("../db/connection");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /api/categories", () => {
+  test("200: returns an array of category objects, each of which has a slug and a description", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body: { categories } }) => {
+        expect(categories).toBeInstanceOf(Array);
```

Comment:
You don't necessarily need this assertion on line 16. You are later using a forEach in the same test which only works on arrays, if this test passes you _know_ it must be an array.